src/providers/BackgroundReplacementProvider/docs/BackgroundReplacementProvider.mdx

```diff
@@ -121,6 +127,20 @@ const MyChild = () => {
     setisVideoTransformCheckOn((current) => !current);
   };
 
+  const changeBackgroundReplacementOption = async (replacementOption: string) => {
```

Comment:
could we change replacementOption to hexColor string such as #ff0000 and pass it down to createColorBlob?